helpers/matrix.yml

```diff
@@ -5,12 +5,12 @@ ES_SUITE:
   - default
   - multi
   - security
-  - 7-alpha
+  - 7-beta
```

Comment:
I think we can just rename this to 7.x already now that it is in the beta phase.

Reply:
`7.0.0` is in beta, `7.x` is a branch targeting `7.1.0` in the elastic stack projects. So I think `7.x` may cause some confusion if it is used to target `7.0.0-beta1` images.

Perhaps `7.0.x` could go here?

Reply:
Pushed rename to `7.0.x`.